tests-legacy/Integration/PrestaShopBundle/Controller/Api/StockManagementControllerTest.php

```diff
@@ -214,7 +214,7 @@ private function assertResponseHasTotalPages($parameters, $expectedTotalPages)
         $this->assertTrue($headers->has('Total-Pages'), 'The response headers should contain the total pages.');
         $this->assertSame(
             $expectedTotalPages,
-            $headers->get('Total-Pages'),
+            (int) $headers->get('Total-Pages'),
```

Comment:
This is necessary for tests to pass ?

Reply:
Yes, otherwise there is datatype mismatch.